### PR TITLE
fix: small screen words breaking, table & code too large

### PR DESCRIPTION
**Pull request by Anouchka-M**
#1813

### CODE REVIEW

**Comment by Anouchka-M** [comment]
@btrd @guillaume-sc a draft so you may check the suggestion described in the issue comment

File: src/app/styles/index.postcss.css [hunk 1]
```diff
@@ -122,6 +122,10 @@ article table {
   @apply border border-sc-gray-3 table-auto w-full mb-4 text-sc-text-4 text-sc-gray-1;
 }
 
+article .table-wrapper {
```
Comment (guillaume-sc) [inline src/app/styles/index.postcss.css:125]: The `table-wrapper` identifier is too generic IMHO. It doesn't convey the idea its content is allowed to overflow horizontally. If we end up having multiple configurations for wrapping enclosed tables, we'll hardly differentiate them if the first one is simply named "wrapper".

I suggest to name it after the behavior it introduces: `overflow-horizontal-content`, or `overflow-horizontal-content-for-XXX` where XXX relates to `w-[95vw]`.

Comment (btrd) [reply src/app/styles/index.postcss.css:125]: What if we apply this to all table inside article ? It simplify things

**Comment by btrd** [comment]
@Anouchka-M Yes I deleted it because the answer was in the issue in your last comment

File: src/app/styles/index.postcss.css (hunk 1 above)
Comment (guillaume-sc) [reply src/app/styles/index.postcss.css:125]: _Disclaimer: I haven't browsed the current source code. Please comment, and disregard my suggestion if it doesn't follow the current code style, or convention._

This is my suggestion, indeed, so it doesn't add one level of indirection for the reader before understanding what it is. The class name `table-wrapper` gives me the place in the HTML code, however at the moment I read it I'm already in the HTML code. It gives me the identifier to look for in the CSS, though this is the indirection I was mentioning.

Comment (Anouchka-M) [reply src/app/styles/index.postcss.css:125]: @btrd What do you think about this solution and about the class naming ?

**Comment by Anouchka-M** [comment]
Some `table` elements are on plain html, and others on markdown.
Had to found a solution to add a `div` and apply the class on markdown ones. Found something but I think it's not really "clean" to mix html and md like this.
I also don't understand why we have some `table` on md, and others on plain html.

File: src/_posts/platform/internals/stacks/2000-01-01-stacks.md [hunk 2]
```diff
@@ -12,12 +12,14 @@ well-known Linux distribution.
...
 Docker images are open source and can be found on the Docker hub:
 
-{: .table }
+<div class="overflow-horizontal-content" markdown="1">
 | Name | Base Distribution | Supported Until | Docker Hub |
 | ---- | ----------------- | --------------- | ---------- |
 | [scalingo-20]({% post_url platform/internals/stacks/2000-01-01-scalingo-20-stack %}) | Ubuntu 20.04 | April 2025 | [scalingo/scalingo-20](https://hub.docker.com/r/scalingo/scalingo-20) |
 | [scalingo-18]({% post_url platform/internals/stacks/2000-01-01-scalingo-18-stack %}) | Ubuntu 18.04 | April 2023 | [scalingo/scalingo-18](https://hub.docker.com/r/scalingo/scalingo-18) |
 | [scalingo-14]({% post_url platform/internals/stacks/2000-01-01-scalingo-14-stack %}) | Ubuntu 14.04 | December 2019 | [scalingo/scalingo-14](https://hub.docker.com/r/scalingo/scalingo-14) |
+{: .table }
```
Comment (guillaume-sc) [inline src/_posts/platform/internals/stacks/2000-01-01-stacks.md:21]: Why does this move _after_ the Markdown table?

**Review by guillaume-sc — COMMENTED** [review]
I read your concern regarding mixing up Markdown and HTML, however I consider HTML comes to the rescue where Markfown falls short of options.

Can you confirm why there's a `{: .table}` moving after the actual Markdown table? I have no other comment, and I'll approve after you shed light on this point.

File: src/_posts/platform/internals/stacks/2000-01-01-stacks.md (hunk 2 above)
Comment (Anouchka-M) [reply src/_posts/platform/internals/stacks/2000-01-01-stacks.md:21]: Yes @guillaume-sc , I don't know why but it wasn't working if leaving it up: the "implicit" html under markdown wasn't interpreted so it was just plain text.

**Review by guillaume-sc — APPROVED** [review]

**Pull request merged**